cmd/kubeadm/app/apis/kubeadm/fuzzer/fuzzer.go

```diff
@@ -79,6 +79,7 @@ func Funcs(codecs runtimeserializer.CodecFactory) []interface{} {
 							Enabled: utilpointer.BoolPtr(false),
 						},
 					},
+					RotateCertificates: true,
```

Comment:
I'm confused by this function. It doesn't look like it's fuzzing anything, just hardcoding the values?

Reply:
I _think_ it's making sure these values from the fuzzer / these defaults are preserved between conversions using the different API versions.

Reply:
I was looking at this a while back, and this overrides the default fuzzing for the attributes and hardcodes specific values. We should most likely only be setting values for attributes here if we need to handle cases where randomized fuzzing would generate invalid configs (attributes that are mutually exclusive, dependent on other attributes, etc).

Reply:
@detiber can you open an issue for improving that? I guess we're setting this for a bit too many items then...